app/vmui/packages/vmui/src/utils/storage.ts

```diff
@@ -1,48 +1,116 @@
+const STORAGE_PREFIX = "VMUI:" as const;
+
 /**
  * Do not use this type in local storage type
  * @deprecated
  * */
-type DeprecatedStorageKeys = "QUERY_HISTORY" | "QUERY_FAVORITES";
-
-export type StorageKeys = "AUTOCOMPLETE"
-  | "NO_CACHE"
-  | "QUERY_TRACING"
-  | "SERIES_LIMITS"
-  | "LEGEND_AUTO_COLLAPSE"
-  | "TABLE_COMPACT"
-  | "TIMEZONE"
-  | "DISABLED_DEFAULT_TIMEZONE"
-  | "THEME"
-  | "EXPLORE_METRICS_TIPS"
-  | "METRICS_QUERY_HISTORY"
-  | "SERVER_URL"
-  | "RAW_JSON_LIVE_VIEW"
-  | "POINTS_SHOW_ALL"
-  | DeprecatedStorageKeys;
-
-
-export const saveToStorage = (key: StorageKeys, value: string | boolean | Record<string, unknown>): void => {
-  if (value) {
-    // keeping object in storage so that keeping the string is not different from keeping
-    window.localStorage.setItem(key, JSON.stringify({ value }));
-  } else {
-    removeFromStorage([key]);
-  }
-  window.dispatchEvent(new Event("storage"));
+const DEPRECATED_STORAGE_KEYS = ["QUERY_HISTORY", "QUERY_FAVORITES"] as const;
```

Comment:
Maybe,should we delete `DEPRECATED_STORAGE_KEYS`? We changed it to new keys half a year ago. Everyone who uses the vmui already migrated to the new format. What do you think?